packages/iov-keycontrol/src/keyring.spec.ts

```diff
@@ -126,6 +126,50 @@ describe("Keyring", () => {
         new Keyring(serialized);
       }).not.toThrow();
     });
+
+    it("throws when the same identity is added twice", async () => {
+      const wallet1 = Ed25519HdWallet.fromMnemonic(
+        "melt wisdom mesh wash item catalog talk enjoy gaze hat brush wash",
```

Comment:
Extracting this into a variable would make it clearer it's the same in both cases.

Reply:
done